src/game_message.cpp

```diff
@@ -324,10 +326,12 @@ static Game_Message::ParseParamResult ParseParamImpl(
 		++iter;
 	}
 
-	// RPG_RT will replace varible substitutions that result in 0
-	// with 1 to avoid invalid actor crash.
+	// Actor 0 references the first party member
 	if (upper == 'N' && value == 0 && got_valid_number) {
-		value = 1;
+		auto* party = Main_Data::game_party.get();
+		if (party->GetBattlerCount() > 0) {
+			value = (*party)[0].GetId();
```

Comment:
Nice catch! All of my tests had Alex as the first party member so I missed this one.

What does it do if the party is empty and you ask for `\n[0]`?

Reply:
I tested this :). Gives the typical "Invalid party members specified". I implemented this by falling through with 0 which gives our typical "Invalid actor" warning.